Sources/Sentry/SentryProfiler.mm

```diff
@@ -36,13 +36,39 @@
 
 static const int kSentryProfilerFrequencyHz = 101;
 
+} // namespace
+
+#    pragma mark - Public
+
 void
-_sentry_configureContinuousProfiling(SentryOptions *options)
+sentry_reevaluateSessionSampleRate(float sessionSampleRate)
+{
+    // TODO: carry over computed sample decision from last launch profile configuration, if there
```

Comment:
`m`: should we resolve this TODO before merge?

Reply:
fixed in https://github.com/getsentry/sentry-cocoa/pull/5008